iis-wf/iis-wf-primary/src/test/resources/eu/dnetlib/iis/wf/primary/processing/sampledataproducer/output/report/primary-processing-execution-times.json

```diff
@@ -1,7 +1,7 @@
 {"key": "processing.documentClassification.duration", "type": "DURATION", "value": "$(LONG_RANGE: [1000, 170000])"}
 {"key": "processing.citationMatching.direct.duration", "type": "DURATION", "value": "$(LONG_RANGE: [1000, 150000])"}
 {"key": "processing.citationMatching.fuzzy.duration", "type": "DURATION", "value": "$(LONG_RANGE: [1000, 900000])"}
-{"key": "processing.citationMatching.total.duration", "type": "DURATION", "value": "$(LONG_RANGE: [1000, 1500000])"}
```

Comment:
Let's drop `total` here (instead of renaming to `deduped`) as already justified in one of the comments above.